Dockerfile

```diff
@@ -10,33 +10,27 @@ RUN mkdir /sctools/
 
 COPY . /sctools 
 
-ARG libStatGen_version="1.0.14"
 ARG htslib_version="1.13"
 
 RUN cd /sctools/fastqpreprocessing &&\
-    wget https://github.com/statgen/libStatGen/archive/v${libStatGen_version}.tar.gz &&\
+    wget https://github.com/khajoue2/libStatGen/archive/refs/tags/v1.0.15.broad.tar.gz &&\
```

Comment:
Is this fork publicly visible? Can we make a PR to the original code base?

Reply:
Yes, it is visible. The real issue is the patches and I don't think they want those changes!